AllASPFuncs.py

```diff
@@ -643,20 +613,17 @@ def constructImages(exposures, ra, dec, size=7, background=False,
 
         bgflux.append(bg)
 
-        #If we are not fitting the background we manually subtract it here.
+        # If we are not fitting the background we manually subtract it here.
         if not background and not truth == 'truth':
-            im -= image._get_header()['SKY_MEAN']
+            image_cutout._data -= image_cutout._get_header()['SKY_MEAN']
```

Comment:
I think it would be clearer to make the if above when you calculate `bg`, and then this data array operation can just simply be

```
image_cutout._data -= bg
Lager.debug(...)
```